6-TypeScript/server/tests/get-test.spec.ts

```diff
@@ -0,0 +1,36 @@
+import testkitCreator from "./testkit";
+import { Chance } from "chance";
+import { UserTodos } from "../models";
+
+describe("test server", () => {
+  const chance = Chance();
+  const testkit = testkitCreator();
+  testkit.beforeAndAfter();
+
+  const { appdriver, dbdriver } = testkit.drivers();
+  it("should return a status code of not found", async () => {
+    const fakeTaskId: string = chance.guid();
+    try {
+      await appdriver.getTaskById(fakeTaskId);
+    } catch (error) {
+      expect(error.response.status).toEqual(400);
+    }
+  });
+  it("should return todo's list", async () => {
+    const userId = chance.guid();
+    let todoId: string, todoId2: string;
+    try {
+      todoId = await dbdriver.addTodo(userId, "FINISH");
```

Comment:
likes the examples ;)
but use chance :)
you're checking that your system works for specific tests and you want to verify it works for everything...

for example - your implementation could be 
```ts
addTodo("finish")
addTodo("checking pr")
```
and your test would be green (when of course - your system should support also `addTodo("say good morning to Tal") ")